### ISSUE TITLE
Issues with chef-zero 5+

### ISSUE THREAD

**Issue by jaredklewis** [thbishop/knife-solo_data_bag#39]
I was experiencing issues reading my data bags. All commands returned the following error:

```
ERROR: The object you are looking for could not be found
Response: Object not found: chefzero://localhost:8889/data/my_app/my_item
```

Adding chef-zero to my Gemfile and pinning it to v4.9 solved the issue.

**Comment by danielcbright** [comment]
👍 I'm experiencing the exact same issue

**Comment by jwmarshall** [comment]
FWIW downgrading to 4.9 did not work for me. It seems like there may be an issue with the --data-bag-path option. If I remove that option it expects the data_bags to be in './data_bags' and if I'm at that path or create a symlink it works fine. For now I've simply updated my Rakefile to change directories before executing the commands.

**Comment by kukido** [comment]
The command works for me, if `data_bag_path` is set in `~/.chef/knife.rb`, with `--data-bag-path` specified on command line, I get the error above.

**Comment by joeyates** [comment]
I've released version 2.1.0 after merging #40.
Please check whether that fixes this issue.

**Comment by joeyates** [comment]
Closing as the issue should be fixed in 2.1.0
